lib/beaker-pe/install/pe_utils.rb

```diff
@@ -891,6 +891,245 @@ def config_hosts_for_proxy_access hosts
           end
         end
 
+        # removes the install_puppet_agent_pe_promoted_repo_on method
+
+        # Install shared repo of the puppet-agent on the given host(s).  Downloaded from
```

Comment:
This rubydoc block I think got separated from its method.

Reply:
I think you deleted this rubydoc block rather than reattaching it to it's method install_puppet_agent_pe_promoted_repo_on, I believe.